pkgs/development/mobile/androidenv/sources.nix

```diff
@@ -148,4 +148,15 @@ in
     };
   };
 
+  source_26 = buildSource {
```

Comment:
Seems superseded in current master branch: https://github.com/NixOS/nixpkgs/blob/d7a68e9bce3ceb302aa12e4339cce119575cbd33/pkgs/development/mobile/androidenv/generated/packages.nix#L1840-L1850